cdap-app-templates/cdap-etl/hydrator-test/src/main/java/co/cask/cdap/etl/mock/batch/LookupTransform.java

```diff
@@ -65,6 +68,29 @@ public void initialize(TransformContext context) throws Exception {
   @Override
   public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter) throws Exception {
     T lookedUpValue = lookup.lookup((String) input.get(config.lookupKey));
+    R lookedUpValueBytes = lookup.lookup(((String) input.get(config.lookupKey)).getBytes());
+    if (lookedUpValue instanceof Result) {
```

Comment:
this is really brittle and ties this class directly to the test case. lookup type and schema should be made into optional config properties.